docs/Technical-Documentation/error-categories-guide.md

```diff
@@ -0,0 +1,82 @@
+# Error Categories Guide
```

Comment:
@reitermb if this is for future reference for the development team, it might be better to refer them to [this directory](https://github.com/raft-tech/TANF-app/tree/develop/tdrs-backend/tdpservice/parsers), for the most current logic around pre-parsing.  Therein, searching for `preparsing_validators` within the directory will point tech team to logic by schema and record type. 

Also, Is there a specific UX mural that tracks pre-parsing details?

Reply:
i meant to make this comment under cat 1 😄

Reply:
I don't think we have a mural—or at least one that's up to date when it comes to definition of parity & error language—but I'm updating this to point to the directory!